R/grade_learnr.R

```diff
@@ -153,12 +165,22 @@ grade_learnr <- function(label = NULL,
       }
     }
 
-  ret <- list(
+  feedback(
     message = checked_result$message,
     correct = checked_result$correct,
     type = message_type,
     location = "append"
   )
+}
+
+
+feedback <- function(message, correct, type, location) {
```

Comment:
Export this and `is_feedback()`

Reply:
Add checkmate checks for all the args

Reply:
I ended up calling this `grade_feedback()` which could supercede `graded()` (https://github.com/rstudio-education/gradethis/pull/123/commits/c449343cbc8696a280b45728604b4213c36e6129). 

@schloerke, do you have an argument for not binding the `graded()` object more closely to learnr feedback? (to me, in this case, it doesn't seem worth decoupling the grade from feedback)

Reply:
After discussion, we're thinking `graded()` should not be deprecated, and `grade_feedback` should have the implementation:

```r
grade_feedback <- function(grade, type, location) {
  if (!is_not_grade(grade)) error
  list(
    message = grade$message,
    correct = grade$correct,
    type = type
    location = location
  )
}
}


```